form/form.go

```diff
@@ -685,6 +711,9 @@ func Builder(visited map[string]struct{}, root bool, fieldname string, help stri
 		s = append(s, `s += '<a href="#" class="del-child btn btn-danger btn-xs" role="button" fieldname="`+fieldname+`">Remove</a>'`)
 		s = append(s, `s += '</div><div class="col-sm-10">'`)
 		s = append(s, `s += '<label class="heading">`+fieldname+`</label>'`)
+		if len(msg.OneofDecl) > 0 {
+			s = append(s, `s += '<br/>This message has `+fmt.Sprintf("%d", len(msg.OneofDecl))+` oneof declarations.<br/>'`)
```

Comment:
I'm not sure, I wanted to include an explanation. But maybe it's better to put some time in the layout itself so the user can easily identify which `oneof` fields belong together.